### PR TITLE
gupnp, gssdp 1.6.0

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by danielnachun** [comment]
`gssdp` needs to be updated to 1.6 as well.

**Comment by carlocab** [comment]
`gupnp-tools` seems to not be compatible with the new `gupnp`/`gssdp`.

File: Formula/gssdp.rb [hunk 1]
```diff
@@ -51,10 +47,10 @@ def install
       -I#{gettext.opt_include}
       -I#{glib.opt_include}/glib-2.0
       -I#{glib.opt_lib}/glib-2.0/include
-      -I#{include}/gssdp-1.2
+      -I#{include}/gssdp-1.6
```
Comment (SMillerDev) [inline Formula/gssdp.rb:50]: Shouldn't this be `#{version.major_minor}`?

Comment (carlocab) [reply Formula/gssdp.rb:50]: Yea, maybe. Never really sure with GNOME. This is meant to be the API version, and sometimes it seems to be the same as the formula version (e.g. here), and sometimes it's not (e.g. glib).

File: Formula/gupnp-tools.rb [hunk 2]
```diff
@@ -22,18 +23,16 @@ class GupnpTools < Formula
   depends_on "gtksourceview4"
   depends_on "gupnp"
   depends_on "gupnp-av"
-  depends_on "libsoup@2"
+  depends_on "libsoup"
```
Comment (cho-m) [inline Formula/gupnp-tools.rb:26]: Can try to backport https://gitlab.gnome.org/GNOME/gupnp-tools/-/commit/7ee5fd9beb04cdb5ebf76b3bf79b132aaacc5e11

**Pull request closed without merging**